signing_clients/apps.py

```diff
@@ -292,6 +292,13 @@ def make_signed(self, signature, outpath=None):
                 # advantage of Firefox's optimized downloading of XPIs
                 zout.writestr("META-INF/zigbert.rsa", signature)
                 for f in sorted(zin.infolist()):
+                    # Make sure we exclude any of our signature and manifest
+                    # files
+                    if f.filename in ("META-INF/manifest.mf",
+                                      "META-INF/zigbert.sf",
+                                      "META-INF/zigbert.rsa",
+                                      "META-INF/ids.json"):
+                        continue
```

Comment:
this seems important not to regress - can you add a test? I think it could be a simple test that calls ``make_signed(...)`` directly instead of an end to end test with proper signatures, etc.